app-deployment.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert
+    kompose.version: 1.21.0 (992df58d8)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: app
+  name: app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: app
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        kompose.cmd: kompose convert
+        kompose.version: 1.21.0 (992df58d8)
+      creationTimestamp: null
+      labels:
+        io.kompose.service: app
+    spec:
+      containers:
+      - args:
+        - /bin/bash
+        - /srv/app/bin/app.sh
+        env:
+        - name: CELERY_BROKER_URL
+          value: redis://broker:6379/0
+        - name: CELERY_TASK_ALWAYS_EAGER
+          value: "False"
+        - name: CONDA_ENVS_PATH
+          value: /srv/cache/miniconda/envs:/opt/miniconda/envs
+        - name: DEBUG
+          value: "False"
+        - name: PIP_DOWNLOAD_CACHE
+          value: /srv/cache/pip
+        - name: POSTGRES_PASSWORD
+          value: mysecretpassword
+        - name: PYTHONWARNINGS
+          value: ignore::DeprecationWarning
+        - name: SECRET_KEY
+          value: ghv8us2587n97dq&w$c((o5rj_$-9#d-8j#57y_a9og8wux1h7
+        - name: bower_storage__packages
+          value: /srv/cache/bower
+        image: parisson/timeside:latest-dev
+        imagePullPolicy: ""
+        name: app
+        ports:
+        - containerPort: 8000
+        resources: {}
+        volumeMounts:
+        - mountPath: /srv/app
+          name: app-claim0
+        - mountPath: /srv/lib/timeside/timeside
+          name: app-claim1
+        - mountPath: /srv/lib/plugins
+          name: app-claim2
+        - mountPath: /etc/apt/sources.list
```

Comment:
It seems that you can't mount a single file like this.

https://stackoverflow.com/questions/33415913/whats-the-best-way-to-share-mount-one-file-into-a-pod
https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath

Reply:
For a first run, I think that you should be able to start the container without mounting a custom sources.list anyway.

@yomguy can you confirm ? What's the point of a custom sources.list here ?